src/gelmandiag.jl

```diff
@@ -83,7 +83,7 @@ function gelmandiag_multivariate(
 
     rfixed = (niters - 1) / niters
     rrandomscale = (nchains + 1) / (nchains * niters)
-    multivariate = rfixed + rrandomscale * LinearAlgebra.eigmax(W \ B)
+    multivariate = rfixed + rrandomscale * eigmax(W \ B)
```

Comment:
```suggestion
    multivariate = rfixed + rrandomscale * LinearAlgebra.eigmax(W \ B)
```

Reply:
just tried adding this back but even though LinearAlgebra is obviously in the manifest it won't allow `LinearAlgebra.eigmax` to be explicitly scoped:
```
julia> gelmandiag_multivariate(tchain)
ERROR: UndefVarError: LinearAlgebra not defined
Stacktrace:
 [1] gelmandiag_multivariate(chains::Array{Float64, 3}; kwargs::Base.Iterators.Pairs{Symbol, Float64, Tuple{Symbol}, NamedTuple{(:alpha,), Tuple{Float64}}})
   @ MCMCChains ~/code/MCwork/dev/MCMCChains/src/gelmandiag.jl:86
```